tests/e2e/expected_results/slsa-verifier/slsa-verifier_explicitly_provided_cue_PASS.json

```diff
@@ -1,50 +1,50 @@
 {
     "metadata": {
-        "timestamps": "2024-05-07 15:16:38",
+        "timestamps": "2024-05-14 13:23:14",
         "has_passing_check": true,
         "run_checks": [
-            "mcn_build_script_1",
-            "mcn_build_service_1",
-            "mcn_provenance_derived_commit_1",
-            "mcn_trusted_builder_level_three_1",
+            "mcn_provenance_witness_level_one_1",
             "mcn_provenance_derived_repo_1",
             "mcn_build_as_code_1",
             "mcn_provenance_available_1",
-            "mcn_infer_artifact_pipeline_1",
-            "mcn_provenance_expectation_1",
             "mcn_version_control_system_1",
-            "mcn_provenance_witness_level_one_1"
+            "mcn_provenance_expectation_1",
+            "mcn_trusted_builder_level_three_1",
+            "mcn_infer_artifact_pipeline_1",
+            "mcn_provenance_derived_commit_1",
+            "mcn_build_script_1",
+            "mcn_build_service_1"
         ],
         "check_tree": {
             "mcn_provenance_available_1": {
-                "mcn_provenance_expectation_1": {},
+                "mcn_provenance_level_three_1": {},
                 "mcn_provenance_witness_level_one_1": {},
-                "mcn_provenance_level_three_1": {}
+                "mcn_provenance_expectation_1": {}
             },
             "mcn_provenance_derived_commit_1": {},
             "mcn_version_control_system_1": {
-                "mcn_build_script_1": {},
                 "mcn_trusted_builder_level_three_1": {
                     "mcn_build_as_code_1": {
-                        "mcn_build_service_1": {},
-                        "mcn_infer_artifact_pipeline_1": {}
+                        "mcn_infer_artifact_pipeline_1": {},
+                        "mcn_build_service_1": {}
                     }
-                }
+                },
+                "mcn_build_script_1": {}
             },
             "mcn_provenance_derived_repo_1": {}
         }
     },
     "target": {
         "info": {
-            "full_name": "pkg:github.com/slsa-framework/slsa-verifier@fc50b662fcfeeeb0e97243554b47d9b20b14efac",
+            "full_name": "pkg:github.com/slsa-framework/slsa-verifier@6fb4f7e2dd9c2f5d4f55fa88f6796278a7bba6d6",
             "local_cloned_path": "git_repos/github_com/slsa-framework/slsa-verifier",
             "remote_path": "https://github.com/slsa-framework/slsa-verifier",
-            "branch": "main",
-            "commit_hash": "fc50b662fcfeeeb0e97243554b47d9b20b14efac",
-            "commit_date": "2022-10-04T01:00:02+00:00"
+            "branch": null,
+            "commit_hash": "6fb4f7e2dd9c2f5d4f55fa88f6796278a7bba6d6",
+            "commit_date": "2022-08-25T11:37:20-05:00"
         },
         "provenances": {
-            "is_inferred": true,
+            "is_inferred": false,
```

Comment:
This is not related to this Pull Request :sweat_smile:. 
It looks like even though we provide a provenance as input, it's not displayed here in the report, but rather the default inferred provenance content instead. I believe this is because we haven't tidied up different places where provenances are stored.